### ISSUE TITLE
Understanding SliceLayer and ColWiseMultLayer

### ISSUE THREAD

**Issue by mantisboxer** [theislab/dca#5]
Gokcen,

I'm trying to implement you concepts for a different (and far simpler) application and your jupyter notebook and DCA code have been very helpful. I have been able to reproduce your toy results fairly easily with a few exceptions...(I'm working in R Keras, which has a few limitations). 

Can you help me understand a few things....

1. Is the ColWiseMultLayer different than using keras.layers.Multiply() ?

2. What is the SliceLayer doing exactly? Is it different than simply concatenating the input tensors? The loss functions don't seem to be splitting the incoming tensors in any way.

I'm currently running as a multiple output model, but I'm not sure how to deal with the scale mismatch of the loss components.

Thanks!
